src/server/controllers/buzz.controller.ts

```diff
@@ -378,3 +380,20 @@ export const claimDailyBoostRewardHandler = async ({ ctx }: { ctx: DeepNonNullab
     throw parsedError;
   }
 };
+
+export function getDailyBuzzPayoutHandler({
+  input,
+  ctx,
+}: {
+  input: GetDailyBuzzCompensationInput;
+  ctx: DeepNonNullable<Context>;
+}) {
+  if (!ctx.user.isModerator) input.userId = ctx.user.id;
+  if (!input.userId) input.userId = ctx.user.id;
+
+  try {
+    return getDailyBuzzPayoutByUser({ userId: ctx.user.id, ...input });
```

Comment:
Overall same comment with the naming here. We'll likely have Multiple daily buzz payouts as time go by. (my guess/expectation) - so we might wanna clear out that this is genreation related, or creator compensation. Something like that